dao/src/integ-test/java/uk/gov/moj/sdt/dao/TargetApplicationDaoTest.java

```diff
@@ -1,80 +1,70 @@
 import uk.gov.moj.sdt.test.utils.TestConfig;
+import uk.gov.moj.sdt.utils.SpringApplicationContext;
 
 import javax.persistence.criteria.CriteriaBuilder;
 import javax.persistence.criteria.CriteriaQuery;
 import javax.persistence.criteria.Root;
 
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 /**
  * Test {@link TargetApplicationDao} query methods.
  *
  * @author Mark Dathorne
  */
 @ActiveProfiles("integ")
-@RunWith(SpringRunner.class)
 @SpringBootTest(classes = { TestConfig.class, DaoTestConfig.class})
+@ExtendWith(SpringExtension.class)
 @Sql(scripts = {"classpath:uk/gov/moj/sdt/dao/sql/TargetApplicationDaoTest.sql"})
 @Transactional
-public class TargetApplicationDaoTest extends AbstractIntegrationTest {
+class TargetApplicationDaoTest extends AbstractIntegrationTest {
 
-    /**
-     * Logger object.
-     */
-    private static final Logger LOGGER = LoggerFactory.getLogger(TargetApplicationDaoTest.class);
     CriteriaBuilder criteriaBuilder;
     CriteriaQuery<TargetApplication> criteriaQuery;
     Root<TargetApplication> root;
 
-    private static final String EXISTING_TARGET_APPLICATION_CODE = "1104";
-    private static final String NON_EXISTENT_TARGET_APPLICATION_CODE = "AX04";
+    private static final String EXISTING_TARGET_APPLICATION_CODE = "MCOL";
+    private static final String NON_EXISTENT_TARGET_APPLICATION_CODE = "UNKNOWN";
 
-    @Before
+    @BeforeEach
     public void setup() {
-        final ITargetApplicationDao targetApplicationDao = this.applicationContext.getBean(ITargetApplicationDao.class);
+        final ITargetApplicationDao targetApplicationDao = SpringApplicationContext.getBean(ITargetApplicationDao.class);
         criteriaBuilder = targetApplicationDao.getEntityManager().getCriteriaBuilder();
         criteriaQuery = criteriaBuilder.createQuery(TargetApplication.class);
         root = criteriaQuery.from(TargetApplication.class);
     }
 
     @Test
     void successfullyGetTargetApplicationForExistingCode() {
-        final ITargetApplicationDao targetApplicationDao = this.applicationContext.getBean(ITargetApplicationDao.class);
+        final ITargetApplicationDao targetApplicationDao = SpringApplicationContext.getBean(ITargetApplicationDao.class);
 
         final ITargetApplication targetApplication = targetApplicationDao.getTargetApplicationByCode(EXISTING_TARGET_APPLICATION_CODE);
-        if (targetApplication != null) {
-            LOGGER.debug("Retrieved target application for code [{}]", targetApplication.getTargetApplicationCode());
-        } else {
-            fail("Could not find target application with code [" + EXISTING_TARGET_APPLICATION_CODE + "]");
-        }
+        assertNotNull(targetApplication,
+                "Could not find target application with code [" + EXISTING_TARGET_APPLICATION_CODE + "]");
```

Comment:
It might also be worth checking the returned targetApplication to ensure the code matches the one being searched for.

Reply:
done